chore: change wording and move periods outside of links (#1703)
clerk/clerk-docs: docs/authentication/social-connections/oauth.mdx

> [!NOTE]
> When using social connections, the sign-up and sign-in flows are equivalent. If a user doesn't have an account and tries to sign in, an account will be made for them, and vice versa.

> The easiest way to add social connections is by using Clerk's [prebuilt components](/docs/components/overview). If you prefer a more custom solution, see the how to [build a completely custom social connection flow](/docs/custom-flows/oauth-connections).

Reviewer: ```suggestion
The easiest way to add social connections is by using Clerk's [prebuilt components](/docs/components/overview). If you prefer building a custom user interface, see [how to build a custom social connection flow using the Clerk API](/docs/custom-flows/oauth-connections).
```

> [!NOTE]
> When using social connections, the sign-up and sign-in flows are equivalent. If a user doesn't have an account and tries to sign in, an account will be made for them, and vice versa.

The easiest way to add social connections to your Clerk app is by using [prebuilt components](/docs/components/overview). If you prefer a more custom solution, see the [build a social connection flow using the Clerk API guide](/docs/custom-flows/oauth-connections).

## Before you start

> - You need to create a Clerk application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out the [Set up your application](/docs/quickstarts/setup-clerk) guide.

Reviewer: ```suggestion
- You need to create a Clerk application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out the [setup guide](/docs/quickstarts/setup-clerk).
```

When signed in, a user can connect to further social providers. There is no need to perform another sign-up.

> When using Clerk's [Account Portal](/docs/customization/account-portal/overview) pages, users can see which providers they have already connected to and which ones they can still connect to on their [user profile page](/docs/customization/account-portal/overview#user-profile).

Reviewer: ```suggestion
When using the [Account Portal](/docs/customization/account-portal/overview) pages, users can see which providers they have already connected to and which ones they can still connect to on their [user profile page](/docs/customization/account-portal/overview#user-profile).
```

Instead of the typical OAuth flow that performs redirects in a browser context, you can utilize Apple's native authorization and provide the openID token and grant code to Clerk. Clerk ensures that the user will be verified in a secure and reliable way with the information that Apple has provided about the user.

> See the [dedicated guide](/docs/authentication/social-connections/apple) for additional information on how to configure Apple as a social provider for your Clerk instance.

Reviewer: ```suggestion
For additional information on how to configure Apple as a social provider for your Clerk instance, see the [dedicated guide](/docs/authentication/social-connections/apple).
```
